sandy/core/cov.py

```diff
@@ -496,11 +496,107 @@ def decompose(self):
         Q, R = scipy.linalg.qr(M.T)
         return R.T
 
-    def glls(self, S, Vy):
+    @classmethod
+    def get_covariance(cls, std):

```

Comment:
this name is unclear...it would be ok if this method was assigned to the FY object.
But it is a CategoryCov method (which is fine).
Then `get_covariance` doesn't mean anything.

Maybe `from_stdev`? or `from_var`?

or maybe you can create both methods